gk/fib.c

```diff
@@ -1254,9 +1259,17 @@ init_grantor_fib_locked(
 		gt_fib = &ltbl->fib_tbl6[fib_id];
 
 	gt_fib->action = GK_FWD_GRANTOR;
-	rte_memcpy(&gt_fib->u.grantor.gt_addr,
-		gt_addr, sizeof(gt_fib->u.grantor.gt_addr));
-	gt_fib->u.grantor.eth_cache = eth_cache;
+	gt_fib->u.grantor.set = rte_malloc("gk_fib.grantor.set",
+		sizeof(*(gt_fib->u.grantor.set)) +
+			sizeof(*(gt_fib->u.grantor.set->entries)), 0);
```

Comment:
We should use `rte_malloc_socket()` instead of `rte_malloc` to preserve the NUMA assumption in the code. The socket should be the same of `gk_conf->lcores[0]`. See `setup_net_prefix_fib()` for a reference.